### PR TITLE
Added systemd-devel CI dependency

### PR DESCRIPTION
**Pull request by lzap**
Unsure about the `;` char at the end, copy-pasted from above.

This syntax kills me.

### CODE REVIEW

File: puppet/modules/slave/manifests/init.pp [hunk 1]
```diff
@@ -146,6 +146,12 @@
         'Debian' => 'libkrb5-dev',
         default  => 'krb5-devel'
       };
+    'systemd-devel':
+      ensure => present,
+      name   => $::osfamily ? {
+        'Debian' => '	libsystemd-dev',
```
Comment (mmoll) [inline puppet/modules/slave/manifests/init.pp]: extra whitespace?

Comment (lzap) [reply puppet/modules/slave/manifests/init.pp]: FIXED

**Comment by mmoll** [comment]
merged, díky @lzap!

**Comment by lzap** [comment]
So I can rebase in 30 minutes, can I?

**Comment by mmoll** [comment]
exactly :+1:

**Pull request merged**